xml/update_caasp.xml

```diff
@@ -0,0 +1,197 @@
+  <para>
+   Version
+  </para>
+ </note>
+ <sect1 xml:id="rook-update-hardware-specs">
+  <title>Recommended hardware specifications</title>
+
+  <para>
+   For &productname; deployed with &rook;, the minimal configuration is
+   preliminary, we will update it based on real customer needs.
+  </para>
+
+  <para>
+   For the purpose of this document, consider the following minimum
+   configuration:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     A highly available &kube; cluster with 3 master nodes
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Four physical &kube; worker nodes, each with two OSD disks and 5 GB of RAM
+     per OSD disk
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Allow additional 4 GB of RAM per additional daemon deployed on a node
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Dual-10 Gb ethernet as bonded network
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     If you are running a hyper-converged infrastructure (HCI), ensure you add
+     any additional requirements for your workloads.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="patch-release-upgrades">
+  <title>Patch release upgrades</title>
+
+  <para>
+   To update a patch release of &rook; to another, you need to update the
+   common resources and the image of the &rook; operator.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Get the latest common resource manifests that contain the relevant changes
+     to the latest version:
+    </para>
+<screen>
+&prompt.user; git clone --single-branch --branch <replaceable>version-number</replaceable> https://github.com/rook/rook.git
+&prompt.user;cd rook/cluster/examples/kubernetes/ceph
+</screen>
+   </step>
+   <step>
+    <para>
+     Apply the latest changes from the next version and update the &rook;
+     Operator image:
+    </para>
+<screen>
+&prompt.kubeuser;kubectl apply -f common.yaml -f crds.yaml
+&prompt.kubeuser;kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:<replaceable>version-number</replaceable>
+</screen>
+   </step>
+  </procedure>
+
+  <note>
+   <para>
+    We recommend updating the &rookceph; common resources from the example
+    manifests before any update. The common resources and CRDs might not be
+    updated with every release, but K8s will only apply updates to the ones
+    that changed.
+   </para>
+  </note>
+ </sect1>
+ <sect1 xml:id="rook-update-procedure">
+  <title>Rook-Ceph Updates</title>
+
+  <para>
+   This is a general guide for updating your &rook; cluster. For detailed
+   instructions on updating to each supported version, refer to the upstream
+   &rook; upgrade documentation:
+   <link xlink:href="https://rook.io/docs/rook/v1.8/ceph-upgrade.html"/>.
+  </para>
+
+  <important>
+   <para>
+    &rook; releases from master are expressly unsupported. It is strongly
+    recommended that you use official releases of &rook;.
```

Comment:
Is this relevant for our docs?

Reply:
@shaas ?

Reply:
As there is no "master" this is not relevant.